src/components/ec-privacy-policy/ec-privacy-policy.vue

```diff
@@ -64,6 +64,10 @@ $ec-privacy-policy-max-width: $body-max-width !default;
 
   &__title {
     @include h4;
+
+    @media screen and (max-width: $ec-privacy-policy-max-width) {
+      display: none;
```

Comment:
we really need that utility-first css library, otherwise I see ourselves writing these kind of css again and again 🤔